distributed/scheduler.py

```diff
@@ -1549,7 +1559,18 @@ def set_nbytes(self, nbytes: Py_ssize_t):
         self._nbytes = nbytes
 
     def __repr__(self):
-        return "<Task %r %s>" % (self._key, self._state)
+        return "<TaskState %r %s>" % (self._key, self._state)
```

Comment:
Also changed the name here so it matches the actual class name, which I have a preference for even though it's a little longer. Though this isn't a strong preference